modules/esaxx/20250106.0/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 8.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@esaxx//:esaxx'
```

Comment:
Please add `--features=parse_headers` and `--process_headers_in_dependencies` to `build_flags` so that we get CI coverage for this header-only library.